core/templates/dev/head/domain/objects/NumberWithUnitsObjectFactorySpec.js

```diff
@@ -133,10 +133,10 @@ describe('NumberWithUnitsObjectFactory', function() {
         NumberWithUnits.fromRawInputString('3# m/s');
       }).toThrow(new Error(errors.INVALID_VALUE));
       expect(function() {
-        NumberWithUnits.fromRawInputString('$3');
+        NumberWithUnits.fromRawInputString('$s 3');
```

Comment:
Remove this test case if it is not necessary.

Reply:
done